README.md

```diff
@@ -19,32 +19,33 @@ via the Cython toolchain. Python in turn gives us the ability to work interactiv
 rich environment of tools for data analysis and scientific computing.
 Furthermore, NetworKit's core can be built and used as a native library if needed.
 
+# Install
 
-## Installation options
+There are three ways to install NetworKit as a Python module.
 
-We support three ways to install NetworKit:
+### Install with pip
 
--   NetworKit Virtual Machine: Download and try NetworKit preinstalled on a virtual machine. This is recommended for users using __Microsoft Windows__.
--   Pip install: Download the NetworKit Python package with pip. This is the easier way to get NetworKit but you can only use NetworKit via Python this way.
--   Build NetworKit from Source: Clone or download the source code of NetworKit and build the C++ and Python modules from source.
-
-More detailed instructions follow after the requirements section.
-With NetworKit as a Python extension module, you get access to native high-performance code and can at the same time work interactively in the Python ecosystem. Although the standard Python interpreter works fine, we recommend [IPython] and [jupyterhub](https://github.com/jupyterhub/jupyterhub) as great environments for scientific computing.
-
-Once you have installed NetworKit, please make sure to check out our [NetworKit UserGuide] for an overview of the features provided in NetworKit.
+    pip3 install [--user] networkit
 
-## Documentation
+### Install with conda-forge
 
-In addition to this `Readme`, the [NetworKit UserGuide] provides an introduction to the NetworKit tools, in the form of an interactive IPython Notebook. The `DevGuide` is meant for developers who would like to contribute. When using NetworKit as a Python module, refer to the docstrings of classes, methods and functions.
+    conda config --add channels conda-forge
+    conda install networkit [-c conda-forge]
 
-C++ sources are also documented in [Doxygen](http://www.stack.nl/~dimitri/doxygen/) format, while the documentation for the Python sources can be generated with [Sphinx](http://sphinx-doc.org/).
-The most recent version of the [documentation can be found online](https://networkit.github.io/dev-docs/index.html). If you have both utilities installed, the documentation can be easily generated by building the `docs` target in your build directory (e.g., `make docs`).
+### Building the Python module from source
 
-## Contact
+    git clone https://github.com/networkit/networkit networkit
+    cd networkit
+    python3 setup.py build_ext [-jX]
+    pip3 install -e .
 
-For questions regarding NetworKit, subscribe to our [e-mail list][list] and feel free to ask.
+The script will call `cmake` and `ninja` (`make` as fallback) to compile
+NetworKit as a library, build the extensions and copy it to the top folder. By
+default, NetworKit will be built with the amount of available cores in
+optimized mode. It is possible the add the option `-jN` the number of threads
+used for compilation.
 
-## Requirements
+# Requirements
```

Comment:
Added a commit with small typo-fixes.

Since the order of install <-> reqs. in this PR with respect to the current Dev-state has changed, I guess this was already considered. Should we place requirements before the install instructions?

Reply:
Yes.